src/services/azblob/backend.rs

```diff
@@ -730,11 +812,172 @@ impl AzblobBackend {
 
         self.client.send_async(req).await
     }
+
+    async fn azblob_batch_del(
+        &self,
+        paths: impl Iterator<Item = &String>,
+    ) -> Result<Response<IncomingAsyncBody>> {
+        let boundary = format!("opendal-{}", Uuid::new_v4());
+        // init batch request
+        let url = format!(
+            "{}/{}?restype=container&comp=batch",
+            self.endpoint, self.container
+        );
+        let req_builder = Request::post(&url).header(
+            CONTENT_TYPE,
+            format!("multipart/mixed; boundary={}", boundary),
+        );
+
+        let mut body = BytesMut::new();
```

Comment:
The whole body construction logic is complex and hard to test. How about extract them to `struct AzblobBatchDeleteRequest` and implement `to_bytes` for this struct?